addons/l10n_ar/models/account_tax_group.py

```diff
@@ -30,3 +31,23 @@ class AccountTaxGroup(models.Model):
         ('8', '5%'),
         ('9', '2,5%'),
     ], string='VAT AFIP Code', index=True, readonly=True)
+
+    @api.ondelete(at_uninstall=False)
+    def check_uninstall_required(self):
+        """
+        Make sure we don't uninstall a required tax group
+        """
+        ar_companies = self.filtered(lambda g: g.company_id.chart_template.startswith('ar_')).mapped('company_id')
+        profits_tax_groups = {
+            self.env['account.chart.template'].with_company(ar_company).ref(
+                'tax_group_percepcion_ganancias',
+                raise_if_not_found=False,
+            ) for ar_company in ar_companies
```

Comment:
probably more efficient to do 1 search on ir.model.data and map :thinking: Did you consider that solution?